.eslintrc.json

```diff
@@ -54,7 +54,8 @@
       {
         "checkIdentifiers": false,
         "checkStrings": false,
-        "checkStringTemplates": false
+        "checkStringTemplates": false,
+        "customWordListFile": "./custom-word-list.txt"
```

Comment:
```suggestion
```

The word needs to be added in cspell.json. 
You can remove the custom-word-list.txt file.